bot.py

```diff
@@ -50,55 +64,41 @@ def get_confirm_message(lang: str):
     return CONFIRMS[lang][key]
 
 
+def detect_language(text: str) -> Optional[str]:
+    language = None
+    for name in AGREE_NAMES.keys():
+        if name in text:
+            language = AGREE_NAMES[name]
```

Comment:
If we found language, we should immediately return. No need to finish the loop.

Reply:
Updated